docs/mining/overview.md

```diff
@@ -26,74 +26,141 @@ transactions waiting to be processed in the mempool.
 
 ---
 
-## <img class="dcr-icon" src="/img/dcr-icons/Download.svg" /> Select and Download Mining Software
+## Solo Mining or Pool Mining
 
-### Official Decred Miner (gominer)
+##### <img class="dcr-icon" src="/img/dcr-icons/Solo.svg" /> Solo Mining
 
-Gominer is the official Decred miner developed and supported by the Decred team. It is the easiest miner to setup and start mining and so it is recommended for most users. Currently, the latest version of gominer is **:octicons-mark-github-16: [v{{ gominerversion }}](https://github.com/decred/gominer/releases/)** and the official binaries can be downloaded from **<https://github.com/decred/decred-binaries/releases/tag/v1.0.0>**
+:fontawesome-solid-exclamation-triangle: **Solo mining is not recommended and is not covered by this documentation!** The Decred network regularly sees a network hash rate of up to 422Ph/s. Solo mining is generally only done by advanced individuals or organized groups with a large cluster of GPUs so it is not addressed here.
 
-Please be certain to select the correct operating system (Windows/Linux) and the correct version for your GPU type (CUDA for NVIDIA cards, OpenCL/OpenCLADL for AMD cards). gominer is only available to 64-bit operating systems. User guides to start mining with gominer can be found here:
 
-- [gominer Pool-Mining](../mining/proof-of-work/pool-mining/gominer.md)
+#### Pool Mining
 
-### Unofficial Miners
+When you mine in a pool, your hashrate is combined with all the other pool miners’ hashrates to look for the correct solution for a block. You will receive a reward based on the amount of work your miner performs in the pool. Most mining pools pay on a pay-per-share method meaning that miners earn a steady amount of Decred rather than the “all or none” of solo mining.
 
-* :octicons-mark-github-16: [cgminer](https://github.com/kR105-zz/cgminer) - cgminer is a popular miner for **AMD** GPUs that has a long history of use in many different cryptocurrencies. It is more difficult to use than the decred gominer.
 
-* :octicons-mark-github-16: [ccminer](https://github.com/tpruvot/ccminer) - ccminer is a popular miner for **NVIDIA** GPUs that has a long history of use in many different cryptocurrencies. It is more difficult to use than the decred gominer.
+## Choosing and configuring a mining pool 
 
-* :octicons-mark-github-16: [sgminer](https://github.com/tpruvot/sgminer) - Those with an **AMD** graphics card running Windows may want to download sgminer.
+To finish the configuration, you will need to choose a mining pool. You can consider payout method ([PPS vs PPLNS](https://www.hashrateindex.com/blog/pps-fpps-pplns-pps_plus)), pool fee, geographic location of the pool, ping time to the server, user interface, statistics / data, customer support, community channels and more. Mining pools all work more or less the same but you may wish to sign up at multiple pools and see which one suits you the best.
```

Comment:
Can we please keep the suggestion to consider a smaller pool, something like:

> You may wish to consider selecting a smaller pool to distribute the hashrate more widely and better decentralize the networks mining power.

Reply:
Added suggestion